src/images/maps/map_points.R

```diff
@@ -28,24 +28,24 @@ box::use(./map_theme)
 map_points <- function(
     iso3,
     df,
-    val_col,
-    size,
+    pt_value_name,
+    pt_value_label,
```

Comment:
yeah i just changed it back so not to affect other scripts. I think it's fine as we are aware how it works, but i think it's still confusing because the `size` argument in `aes()` takes the column, but later inside labs `size` takes a label. I sort of get the logic trying to mirror ggplot, but something feels funky

Confusing elements extracted from original in pseudo code below:

```
geom_sf(
  data= data,
  aes(
    size= .data[val_coll]
  )
)+
  labs(
    size =size
  )

```